test/xcode-specs.js

```diff
@@ -83,4 +83,10 @@ describe('xcode @skip-linux', () => {
     (parseFloat(version)-6.1).should.be.at.least(0);
   });
 
+  it('should get a list of iOS devices', async() => {
+    let devices = await xcode.getConnectedDevices();
+    should.exist(devices);
```

Comment:
Here is an example for those interested. https://gist.github.com/sebv/50c3481ab33ebd9b93cb

Yeah merge it from my side it is ok.